integrations/access/accessrequest/app.go

```diff
@@ -144,9 +153,13 @@ func (a *App) run(ctx context.Context) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-
-	if err := a.initAccessMonitoringRulesCache(ctx); err != nil {
-		return trace.Wrap(err)
+	if len(acceptedWatchKinds) == 0 {
```

Comment:
Should we do this check right after NewJobWithConfirmedWatchKinds call?

Reply:
I don't think so because the watcher only starts after `process.SpawnCriticalJob` and we are only guaranteed to receive the Event OpInit after the `job.WaitReady` returns.
So, only after `job.WaitReady` returns can we check the `acceptedWatchKinds`.